pre_commit_hooks/file_contents_sorter.py

```diff
@@ -18,9 +18,12 @@
 FAIL = 1
 
 
-def sort_file_contents(f: IO[bytes]) -> int:
+def sort_file_contents(f: IO[bytes], key : Optional[callable]) -> int:
```

Comment:
mypy points out this should be `Optional[Callable[[bytes], Any]]`

Reply:
Fixed.